install.js

```diff
@@ -8,6 +8,7 @@ connection.query(
 	'points INT,' +
 	'num_lines INT,' +
 	'level INT,' +
+	'pf VARCHAR(200),' +
```

Comment:
With the latest commits I improved the profile page and login menu. One of the new things is using people their profile pictures, `pf` is for storing those. When their user page is loaded, the website fetches the profile picture and inserts or updates it into the user table.